crates/aptos/src/common/types.rs

```diff
@@ -215,6 +215,8 @@ pub struct ProfileConfig {
     /// URL for the Faucet endpoint (if applicable)
     #[serde(skip_serializing_if = "Option::is_none")]
     pub faucet_url: Option<String>,
+    /// Derivative path index of the account on ledger
+    pub bip44_index: Option<String>,
```

Comment:
I wonder if I should store the whole "derivative path" here? Instead of the account index.

m/44'/637'/{bip44_index}'/0'/0'

Reply:
Could you explain why you'd choose one or the other?

Reply:
Yep good question!
 
I actually prefer the full derivative path here, so I can just use it directly. Though I am not sure if this full derivative path is considered partial "secret".

Reply:
Yeah full derivative path is safer imho, and no it's not secret so all good :)

Reply:
But also it can be easily migrated in case we later on decide to change the path format. So also wouldn't worry too much.
For reference, in Petra we're storing the full path

Reply:
Good call! Will update this to full path